src/js/jsx/Main.jsx

```diff
@@ -148,18 +137,20 @@ define(function (require, exports, module) {
         },
 
         render: function () {
-            var hasDocument = this.getFlux().store("application").getCurrentDocument(),
-                colorStop = this.state.colorStop || this.props.initialColorStop,
+            /* global _spaces */
+            var colorStop = this.state.colorStop || this.props.initialColorStop,
                 stopClass = "color-stop__" + colorStop.toLowerCase();
 
+            // To enable animations, add a feature flags object to manifest.json and then
+            // set the property use_animations:true in that object. You will need to restart Photoshop
+            // jscs:disable requireCamelCaseOrUpperCaseIdentifiers
+            var useAnimations = !!(_spaces.feature_flags && _spaces.feature_flags.use_animations);
```

Comment:
I think this can just be a constant instead of recalculating it on every render?